hapi-fees/src/main/java/com/hedera/services/usage/crypto/CryptoOpsUsage.java

```diff
@@ -178,4 +178,30 @@ public void cryptoCreateUsage(final SigUsage sigUsage,
 		accumulator.addRbs(maxAutomaticTokenAssociations * lifeTime * CREATE_SLOT_MULTIPLIER);
 		accumulator.addNetworkRbs(BASIC_ENTITY_ID_SIZE * USAGE_PROPERTIES.legacyReceiptStorageSecs());
 	}
+
+	public void cryptoApproveAllowanceUsage(final SigUsage sigUsage,
+			final BaseTransactionMeta baseMeta,
+			final CryptoApproveAllowanceMeta cryptoApproveMeta,
+			final ExtantCryptoContext ctx,
+			final UsageAccumulator accumulator) {
+		accumulator.resetForTransaction(baseMeta, sigUsage);
+		accumulator.addBpt(cryptoApproveMeta.getMsgBytesUsed());
+
+		var newVariableSerialBytes = 0;
+		newVariableSerialBytes += cryptoApproveMeta.getAggregatedNftAllowancesWithSerials() * LONG_SIZE;
+		accumulator.addRbs(newVariableSerialBytes);
```

Comment:
Should this line be here? I think all we want to do here is:
1. Compute the total bytes used by the new allowances (call this `S`).
2. Determine the lifetime of this account (call this `T`).
3. Call `accumulator.addRbs(S * T)`.

Reply:
Agreed. Thanks Michael. Will fix that in this [PR](https://github.com/hashgraph/hedera-services/pull/2851/)